backend/trigger_handler.py

```diff
@@ -2,16 +2,20 @@
 The handler of this module will be called once upon every deployment
 """
 
-import logging
 import os
 
+from alembic import command
+from alembic.config import Config
+
 from dataall.base.db import get_engine
 from dataall.base.loader import load_modules, ImportMode
 from dataall.core.permissions.services.tenant_policy_service import TenantPolicyService
 
-logger = logging.getLogger()
-logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
-log = logging.getLogger(__name__)
+
+def db_migrations(event, context) -> None:
+    alembic_cfg = Config('alembic.ini')
+    alembic_cfg.set_main_option('script_location', './migrations')
```

Comment:
Why `script_location` is set programatically here and not loaded from .ini?

Reply:
If I understand this correctly, we are replacing the CLI commands by alembic classes (Config) and packages (Command). So lines 16 and 18 should be equivalent to `'alembic -c backend/alembic.ini upgrade head',` but then line 17 that Sofia pointed out... is it there the same way `'export PYTHONPATH=backend',` was in the CodeBuild stage?

Reply:
The script location is loaded from the ini I overwrite it here.

That's because in the programmatic invocation we call alembic from within the Docker container. When we creating the image we copy the `./backend/*` into `/app` so the path is now `/app/migrations/...` not `/app/backend/migrations/...`.

I didn't change the `.ini` because I wanted to allow devs to invoke alembic manually as they currently do today (via the `Makefile`)